src/AWS.Deploy.CLI/Commands/CommandFactory.cs

```diff
@@ -485,6 +485,7 @@ private Command BuildServerModeCommand()
                 serverModeCommand.Add(new Option<int>(new[] { "--port" }, description: "Port the server mode will listen to."));
                 serverModeCommand.Add(new Option<int>(new[] { "--parent-pid" }, description: "The ID of the process that is launching server mode. Server mode will exit when the parent pid terminates."));
                 serverModeCommand.Add(new Option<bool>(new[] { "--encryption-keyinfo-stdin" }, description: "If set the cli reads encryption key info from stdin to use for decryption."));
+                serverModeCommand.Add(new Option<bool>(new[] { "--no-encryption-keyinfo" }, description: "If set the cli bypasses the need for an encryption key."));
```

Comment:
`--encryption-keyinfo-stdin` and `--no-encryption-keyinfo` are too confusing now given both are bool options. From the description, I couldn't understand what will be the end result.

Not related to this change but
```
"If set the cli reads encryption key info from stdin to use for decryption."
```
"encryption key" looks wrong, either it should be a symmetric or asymmetric key.

Reply:
I would be fine and I think that would meet AppSec requirement if we remove `--encryption-keyinfo-stdin` and always read from stdin unless `--no-encryption-keyinfo` is set. I would also rename `--no-encryption-keyinfo` to be `--unsecure-mode`. Encryption keyinfo is kind of an implementation detail and the point we want to put out is they are choosing to a less secure mode..

Reply:
Fixed